test/input_gen/genInput.py

```diff
@@ -51,6 +51,12 @@ def save(filename, *data):
 def gen_tensor(shape, dtype=dtypes.float32):
     return np.random.randint(1, 10, shape).astype(np.float32)
 
+##
+# @brief generate random tensor
+def gen_tensor_int(shape):
```

Comment:
How is this different from `gen_tensor`? Is this just for max value to be 100?

Reply:
I think you will have to remove int from the gen_tensor  and limit range here to be 10.